src/integration-test/resources/application-it.yml

```diff
@@ -1,11 +1,16 @@
-spring:
-  jpa:
-    hibernate:
-      ddl-auto: validate
 integration:
   messaging:
     base-url: https://api.localhost.se/messaging
     recipient-adress: someadress@sundsvall.se
     token-uri: https://api.localhost.se/oauth/token
     client-id: some-client-id
     client-secret: some-client-secret
+scheduled:
+  initial-delay: PT5S
+  fixed-rate: PT1M
+  email-age-check:
+    initial-delay: PT5S
+    fixed-rate: PT1H
+properties:
+  credentials:
+    secret-key: tY2$9dHiu*%EqRLhgUM8X446AF4*eSWV
```

Comment:
Is this a secret used in test- or production environment? If so, replace in this configuration as repository is public.